chatbot-core/backend/app/main.py

```diff
@@ -1,29 +1,46 @@
+from contextlib import asynccontextmanager
 from fastapi import FastAPI
 
-from app.utils.logger import LoggerFactory
 from app.routers import base
-from app.routers.v1 import embedding_model
-from app.settings.constants import Constants
+from app.routers.v1 import connector
+from app.settings import Constants
+from app.utils.llama_index_configuation import init_llamaindex_config
+from app.utils.logger import LoggerFactory
 
 logger = LoggerFactory().get_logger(__name__)
 
 
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Comment:
Does this function also include session initialization of MSSQL, Qrant, Redis, and MinIO sessions?

Reply:
No, it will only initialize the LlamaIndex configuration like `llm`, `embed_model` or `callback_manager`. If necessary, I will write a function so that all storage sessions / clients will be initialized too.

Reply:
I'm not sure the what should be best practice here... so we can go ahead...